setuptool/setuptool.go

```diff
@@ -324,12 +333,12 @@ func retrieveAccountKey(t *term) {
 
 func addUser(t *term) {
 	userInput(t, "Enter the users' emails> ", config.AddUser)
-	fmt.Println("User added.")
+	fmt.Println("User added.\n")
```

Comment:
why do you need \n when you are doing println? Do you want extra empty line between the outputs?

Reply:
Yep one more line seems clearer=)